src/studio/nuxeo-studio/working-in-studio/migration-services.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Migration Services
+review:
+    comment: ''
+    date: '2018-12-05'
+    status: ok
+tree_item_index: 950
+toc: true
+---
+The JSF UI to Web UI migration tool is a safe and automated way to create, within a few minutes, an equivalent Web UI configuration based on your existing JSF UI configuration.
+
+The migration tool reduces dramatically the effort needed to transition to [Web UI]({{page space="nxdoc" page="web-ui"}}), our user interface that uses modern web standard technologies to provide an outstanding user experience.
+
+Running the migration tool does not delete any data. If you are not satisfied with the result, you can:
+* go back to the state your project was before triggering it anytime,
+* delete the additional configuration that was generated partially or totally.
+
+After migration is run, a detailed report is created in **Migration Services** so that you can check the results.
+
+{{#> callout type='info' heading='part of early access program'}}
+JSF UI to Web UI migration tool is available through our early access program, offering stable features you can co-create with us as they are only partially complete.<br /><br />
+
+- Not a member of it yet? [Learn about its benefits, what's planned and how to get in]({{page page="early-access-program"}}).
+<br /><br />
+- Already a member? Help us co-creating it by [telling us about your experience](https://docs.google.com/forms/d/e/1FAIpQLSdoDfpJOnFwkxHAHOk5aq5KHh9myxF6fISugnMzVmvuQSp-fw/viewform?usp=pp_url) </a>.
+{{/callout}}
+
+## What is migrated?
+
+| Migrated items | Not currently migrated |
+|----------------|-------------------|
+| Document layouts | Content views that contain query parameters or that reference a table layout |
+| Workflow tasks layouts | Translation files  |
+| Form layouts | Tabs     |
+| Content views that do not reference a table layout |     &nbsp;     |
+| User actions |        &nbsp;    |
+
+This scope is subject to change. Feel free to [let us know about your experience](https://docs.google.com/forms/d/e/1FAIpQLSdoDfpJOnFwkxHAHOk5aq5KHh9myxF6fISugnMzVmvuQSp-fw/viewform?usp=pp_url)
+
+## Processing Migration
+
+{{#> callout type='warning' heading='working during migration'}}
+Although it is possible, we recommend not to work while the migration is happening.
+
+Any configuration you would do in that time frame would not be taken into account by the migration tool, and could lead to conflicts if you decide to merge changes afterwards.
+
+{{/callout}}
+
+### Launching Migration
```

Comment:
I would position this section at the real top because we are talking about the way the migration is processed before where to execute it (to make it more logical)

Reply:
I added the sentence : "To process the JSF UI to Web UI migration, access Migration Services from the Configuration menu in Studio Modeler." Otherwise as I structured the page around a big complex task, it would mean putting the migrated/not migrated items at the very bottom.